customer/views.py

```diff
@@ -4,17 +4,17 @@
 
 from django.views import View
 from django.http  import HttpResponse, JsonResponse
-#from django.db    import IntergrityError
-
 from .models      import Customer
 
-#import my_settings
 
 class SignUpView(View):
     def post(self, request):
         data = json.loads(request.body)
 
         try:
+            if Customer.objects.filter(email=data['email']).exists():
+                return JsonResponse({'message':'Already registered email'}, status=400)
```

Comment:
이 메세지는 프론트에서 활용하나요? 메세지코드 컨벤션이 제각각입니다. 확인해주세요.
```suggestion
                return JsonResponse({'message':'EMAIL_ALREADY_EXIST'}, status=400)
```

Reply:
수정 사항 반영하였습니다.